src/Zynga/Framework/Logging/V1/Config/GlobalConfig/Production.hh

```diff
@@ -0,0 +1,26 @@
+<?hh // strict
+
+namespace Zynga\Framework\Logging\V1\Config\GlobalConfig;
+
+use Zynga\Framework\Logging\V1\Adapter\ErrorLog;
+use Zynga\Framework\Logging\V1\Config\Base as ConfigBase;
+use Zynga\Framework\Logging\V1\Level;
+
+class Production extends ConfigBase {
+
+  public function init(): bool {
+    $errorLog = new ErrorLog();
+    $this->addAdapter($errorLog);
+    $this->unsetLogLevel(Level::DEBUG);
```

Comment:
DEBUG will be super noisy in production if we leave it on. I would suggest disabling this on prod.

Reply:
Sorry, same with INFO.